pkg/terraform/workspace.go

```diff
@@ -46,6 +46,20 @@ func WithLogger(l logging.Logger) WorkspaceOption {
 	}
 }
 
+// WithExec sets the executor of Workspace.
+func WithExec(e k8sExec.Interface) WorkspaceOption {
```

Comment:
I think this could be renamed to `WithExecutor` or we can rename the field name `executor` to `exec`.

Reply:
The reason of this naming is that, in this package, I have added another function `WithExecutor` for `WorkspaceStore`. As you know, Go does not allow this redeclaration.

While looking your comments, I realized that, we do not use the `WithExecutor` for `WorkspaceStore` and I deleted this one and I renamed the `WithExec` to `WithExecutor`. However, I think this is not a good solution for this because in future, we can need to use `WithExecutor` for `WorkspaceStore` I wanted to discuss this issue in this thread.

Another example is that, I had to name the Afero option function for `Workspace` with `WithAferoFs` name. Beacuse WithFs and WithFileSystem names are already used.

I do not resolve this conversation. What are your thoughts?

Reply:
We can consider separating them into different packages but that's not really concern of this PR. So, it should be fine then.